### DISCUSSION TITLE
Ricominciare daccapo ma senza perdere il passato.

### DISCUSSION THREAD

**Discussion by Jocman** [indomus/forum#1798] [HUB: Home Assistant]
Ciao a tutti.
Attualmente penso che utilizzo HA solo al minimo delle sue potenzialità; mi piace l'idea di.....controllare casa (o almeno una parte di essa) come se fossi sull' U.S.S. Enterprise, ma onestamente mi ci sto applicando poco rispetto a quello che vorrei, più che altro perché ho alcune priorità da realizzare a casa (non parlo di domotica) e se metto troppa carne al fuoco poi brucio tutto e non mangio nulla…..
In ogni caso, ogni tanto ho anche l’esigenza di implementare qualcosa su HA (vedasi in questo periodo i climatizzatori con SmartIr – anche se sto avendo qualche intoppo), ed in questa occasione mi sono accorto di due cose. La prima è che ogni riavvio HA ci mette una vita a riavviarsi, o almeno così mi pare; se ricordo bene all’inizio non dico fosse immediata la cosa, ma di certo molto più veloce, adesso ad ogni riavvio mi posso tranquillamente andare a fare un caffè nell’attesa.
La seconda, è che mi ritrovo con oltre 140 entità non attive, probabilmente frutto di test, o anche i vecchi cellulari, o chissà che altro, spesso con avvisi del tipo che l’id non è univoco e non posso fare nulla (in primis eliminarle).
A tutto questo, vorrei passare al sistema doppio “SD+SSD”, quindi approfitterei per fare le due cose insieme.
Tuttavia non vorrei perdere tutto quello che già ho domotizzato (e che mi è costato tempo su tempo).
Quello che vi chiedo è: vorrei reinstallare HA da zero, ma c’è un modo di mantenere quello che ho configurato e che utilizzo attualmente?
Mi spiego: dovrò reinstallare HA-OS, DuckDNS, MQQT, e altro. Ma salvando tutti i vari file yaml (prima della reinstallazione) e ricopiandoli dopo la nuova installazione, mi troverò ciò di funzionante che avevo o inevitabilmente mi ritroverei anche tutto il “rudo”, e quindi una faticata inutile?

**Comment by snakuzzo** [comment]
Se hai HA OS non devi fare niente di particolare.
Nativamente è previsto il backup e restore su nuova installazione.
Fine.

**Reply by Saint72** [reply]
Per passare da HA OS a HA su Raspbian si può usare lo stesso il backup?
Immagino di no... vero?
Io ho installato HA OS per comodità ma adesso avrei "quasi" la necessità di usare il raspberry anche per altro.
Mi devo rifare da capo?

**Reply by snakuzzo** [reply]
Potresti decidere di usare sempre HA OS -> [HA Supervised su Docker](https://indomus.it/guide/installare-e-configurare-home-assistant-supervised-con-docker-su-mini-pc-intel-nuc-computer-con-linux-debian/) ed utilizzerai il backup così com'è.
Se invece vuoi passare a Core, non puoi usare il backup così ad occhi chiusi. Puoi aprirlo (tanto è un file zip) e recuperare quanto presente sotto la directory `/config`. Considera però che con Core non avrai gli addon. Se ad esempio hai zigbee2mqtt, un reverse proxy o altro, ti toccherà installarli a parte

**Comment by Jocman** [comment]
Ti ringrazio.
Ma facendo il backup e restore però non mi ritrovo di nuovo con i problemi che sto rilevando adesso?
Forse la domanda è stupida, ma prima di fare qualcosa e rischiare (forse) di perdere tutto, preferisco chiedere....

**Reply by snakuzzo** [reply]
1. Fai REPLY sotto l'ultimo messaggio, non aprire nuovo thread (come hai fatto ora)
2. I problemi non si risolvono facendo tutto da zero. Ti ci metti, guardi i log, e risolvi pezzo per pezzo. Tra le altre cose è anche il miglior modo per cominciare a capirci qualcosa
